recipes/trefide/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "trefide" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jw3132/conda-forge-tar/raw/master/trefide-{{ version }}.tar.gz
+  sha256: e462466d3a21b6c88c2958541aeaf3caf1cb2ee197f488265bf03fc848eaf562
+
+build:
+  number: 0
+  skip: true  # [win, osx]
+
+requirements:
+  build:
+    - python =3.6
```

Assuming you need a compiler you shouldn't constrain the Python version this way (see comment above):
```suggestion
    - python
```

---

Done